packages/strapi-generate-new/lib/resources/files/config/environments/production/response.json

```diff
@@ -8,5 +8,11 @@
   "poweredBy": {
     "enabled": true,
     "value": "Strapi <strapi.io>"
-  }
+  },
+  "hiddenFields": [
```

Comment:
I'm not sure we have to force `SQL` apps  (MySQL, SQLite, Postgres) to have these values by default.

Reply:
I do agree about removing the timestamp ones, those are very useful (That being said SQL use `created_at` and `updated_at`)

Reply:
Ok I just removed the default values from the configs.